### ISSUE TITLE
Bluray support?

### ISSUE THREAD

**Issue by deri82** [ValveSoftware/SteamOS#123]
...

**Comment by jonesn** [comment]
Just want to ping this again. Having SteamOS handle Blurays officially would be a big plus for a living room pc.

At home I have replaced Windows with SteamOS for games but I miss PowerDVD for Bluray playback.

**Comment by obialfred** [comment]
YES! Steam OS needs NATIVE Blu-ray support. Would also be cool to purchase physical blu-ray games and install them on the system. This would be huge.

**Comment by not83** [comment]
This is for bugs not requests. Valve employees have stated they want to
keep this github for bugs only. Try to keep requests to the SteamOS
discussion page on steam community

**Comment by deri82** [comment]
How does this differ from a bug that you dont suport certain hardware by drivers? It doesnt

**Comment by ProfessorKaos64** [comment]
@deri82 , where has it stated that Blu-ray playback is supported? It isn't. Please use the Steam Universe or Steam Client Beta forum groups. What Linux distributions have native support by default? Everyone I have tried since 2004 usually involves some manual setup.
